tests/components/aladdin_connect/test_init.py

```diff
@@ -23,19 +22,19 @@ async def test_unload_entry(hass: HomeAssistant):
         "homeassistant.components.aladdin_connect.cover.AladdinConnectClient.login",
         return_value=True,
     ):
-
-        await async_setup_component(hass, DOMAIN, entry)
+        await hass.config_entries.async_setup(entry.entry_id)
```

Comment:
Looking at this test, it'd almost the same as the last test in this module. I suggest we keep the last test and remove this test here.

Reply:
Yes, the further result of too much copying and too little reviewing :(